doctr/datasets/datasets/base.py

```diff
@@ -20,12 +22,14 @@ class _AbstractDataset:
     def __init__(
         self,
         root: Union[str, Path],
+        convert_to_relative: bool = True,
```

Comment:
We need to add kwargs in `_VisionDataset` constructor, that are then passed to this constructor